server/src/main/java/org/dst/server/base/DstBaseService.java

```diff
@@ -0,0 +1,11 @@
+package org.dst.server.base;
+
+import org.dst.core.KVStore;
+
+public class DstBaseService {
+  public KVStore store;
```

Comment:
private

Reply:
Can‘t private。

if private ，it can‘t be exends’